test/Modules/MPolyQuo.jl

```diff
@@ -106,7 +106,7 @@ end
   A, _ = quo(R, I)
   A1 = FreeMod(A, 1)
   M, _ = quo(A1, [y*A1[1], z*A1[1]])
-  p = free_resolution(M)
+  p = free_resolution(M, length = 11)
   @test iszero(p[10])
```

Comment:
```suggestion
  @test p[10] isa FreeMod
```
being zero is not a mathematical necessity, but depends on the chosen algorithm. The key point is to see that the resolution actually works. 

But: I think, I have made the resolution over quotient rings lazy and infinite by default. Since for the previous implementation we had nothing better than iterative kernel computation, there was no trade-off from that. This seems to have changed now, so there might be further things to correct down the road. We'll see.